docs/en/tutorials/getting-started.rst

```diff
@@ -228,33 +227,196 @@ entity definition:
          * @var string
          */
         protected $name;
+    }
 
-        public function getId()
+When creating entity classes, all of the fields should be protected or private (not public).
```

Comment:
We should probably rephrase to something like:

 > should be `private`, `protected` when strictly needed and very rarely if not ever `public`

I can also add that `protected` and `public` have some performance overhead (minimal, but it is there). Unsure if we can pack it into the article.

Reply:
I don't think that we need to introduce any limitations in getting started at all. Only one that are really limitations of Doctrine. Proper entity modeling could be covered elsewhere.

Reply:
I'm not sure the performance overhead might be interesting for end-users to know.

Reply:
@fesor it's not a limitation (of Doctrine) is it? Just a good recommendation. There's no explanation though, and the performance concerns are additional to something else.

When creating entity classes, fields should be `private`, `protected` when strictly needed and very rarely if not ever `public`, to make sure encapsulation is not broken by needless exposition.